doc/west/repo-tool.rst

```diff
@@ -217,8 +217,8 @@ to the latest manifest revision, and inspects upstream changes.
 
    .. code-block:: console
 
-      $ west init
-      $ west clone
+      west init
```

Comment:
Because ``west init`` will initialize the **current** folder, it would be good to add a preamble to the workflow to explain setting up the development working directory.  For example, should a developer start with an empty folder (e.g., ``~/zephyrproject``) and cd into it before doing this ``west init`` command?

Reply:
I've added something like that, mirroring the instructions in Getting Started.